docs/experiments-cw09b.md

```diff
@@ -60,47 +60,47 @@ nohup target/appassembler/bin/SearchCollection -topicreader Webxml -index lucene
 Evaluation can be performed using `trec_eval` and `gdeval.pl`:
 
 ```
-eval/trec_eval.9.0/trec_eval -m map -m P.30 src/main/resources/topics-and-qrels/qrels.web.51-100.txt run.cw09b.bm25.topics.web.51-100.txt
```

Comment:
It is a bit odd why there is an extra empty line here.
Did you manually change this file?

Reply:
@Peilin-Yang are those parts auto-gen'ed? Perhaps @Chriskamphuis doesn't know about the details of the regression framework...

Reply:
All `experiments-{$collection}.md` files are auto-generated. 
@Chriskamphuis Could you please run `mvn test` locally and see if there is any diff?

Reply:
@Peilin-Yang No diff for me locally. I only changed the the index.md and yaml files. Then I ran test before pushing the changes. Maybe I accidentally added an extra line when checking if everything was correct, but then it should be removed when running ``` mvn test ```  right?